lib/rspec/collection_matchers/have.rb

```diff
@@ -24,6 +24,12 @@ def relativities
         }
       end
 
+      if RUBY_VERSION == '1.9.2'
+        def to_ary
+          [self]
+        end
+      end
```

Comment:
That would help.  I still don't really understand, the issue, so a thorough comment would help :).